tests/yamitranscode.cpp

```diff
@@ -370,7 +382,12 @@ class TranscodeTest
         m_output->output(src);
 
         fps.log();
-
+        if (m_cmdParam.m_encParams.enableprintFrameLatency){
```

Comment:
Could we just handle this VppOutputEncoder's deconstructor ? YamiTranscode do not need know this details